src/main.cpp

```diff
@@ -3,11 +3,28 @@
 #include "Compositor.hpp"
 #include "config/ConfigManager.hpp"
 #include "init/initHelpers.hpp"
+
 #include <iostream>
+#include <iterator>
+#include <vector>
+#include <stdexcept>
+#include <string>
+#include <filesystem>
+
 #ifdef USES_SYSTEMD
-#include <systemd/sd-daemon.h> // for sd_notify
+    #include <systemd/sd-daemon.h> // for sd_notify
 #endif
 
+
+void help() {
+    std::cout << "usage: Hyprland [arg [...]]." << std::endl;
```

Comment:
why are you using `std::endl`? Why flush after every line...?

Reply:
'Kay, I'll change that. Not a big deal.